src/foam/u2/CSSTokens.js

```diff
@@ -146,6 +146,6 @@ foam.CLASS({
     'inputBorderRadius': '4px',
 
     // FONT
-    'font1': `'IBM Plex Sans', sans-serif`
+    'font1': `'Source Sans Pro', 'Sans pro'`
```

Comment:
Shouldn't we remove the IBM font in the othe file?

Reply:
Yes, we can remove IBM Plex